back-end/config/LoaderFactory.js

```diff
@@ -0,0 +1,27 @@
+const AbstractLoader = require('./loader/AbstractLoader');
+
+class LoaderFactory {
+    constructor() {
+        this.loader = undefined;
+    }
+
+    getLoader() {
+        if (this.loader instanceof AbstractLoader) {
+            return this.loader;
+        }
+
+        switch (process.env.STORAGE) {
+            case 'firebase':
+                this.loader = require('./loader/Firebase');
+                break;
+            case 'local':
+            default:
+                this.loader = require('./loader/Filesystem');
+                break;
+        }
+
+        return this.loader;
```

Comment:
I would say this is a `ConfigFactory`, that it loads the config in the background doesn't change the fact that a config is returned.

Goes for all `Loader` references

Reply:
See above :) I am mixing stuff here, will sort it out!

Reply:
Fixed!